src/session_impl.cpp

```diff
@@ -5504,18 +5504,18 @@ namespace aux {
 			s.dht_total_allocations = 0;
 		}
 
-		s.utp_stats.packet_loss = m_stats_counters[counters::utp_packet_loss];
-		s.utp_stats.timeout = m_stats_counters[counters::utp_timeout];
-		s.utp_stats.packets_in = m_stats_counters[counters::utp_packets_in];
-		s.utp_stats.packets_out = m_stats_counters[counters::utp_packets_out];
-		s.utp_stats.fast_retransmit = m_stats_counters[counters::utp_fast_retransmit];
-		s.utp_stats.packet_resend = m_stats_counters[counters::utp_packet_resend];
-		s.utp_stats.samples_above_target = m_stats_counters[counters::utp_samples_above_target];
-		s.utp_stats.samples_below_target = m_stats_counters[counters::utp_samples_below_target];
-		s.utp_stats.payload_pkts_in = m_stats_counters[counters::utp_payload_pkts_in];
-		s.utp_stats.payload_pkts_out = m_stats_counters[counters::utp_payload_pkts_out];
-		s.utp_stats.invalid_pkts_in = m_stats_counters[counters::utp_invalid_pkts_in];
-		s.utp_stats.redundant_pkts_in = m_stats_counters[counters::utp_redundant_pkts_in];
+		s.utp_stats.packet_loss = std::uint64_t(m_stats_counters[counters::utp_packet_loss]);
```

Comment:
I didn't do that because `utp_status` is deprecated and it's not obvious to me if braking ABI in deprecated API is a good idea. You tell me.